### ISSUE TITLE
Add after-reboot command

### ISSUE THREAD

**Issue by dannyroberts** [dimagi/commcarehq-ansible#1235]
Make
```
commcare-cloud <env> after-reboot <group>
```
an alias for
```
commcare-cloud <env> ansible-playbook deploy_stack.yml --tags=after-reboot --limit=<group> --skip-check
```

**Comment by esoergel** [comment]
One thing to note is that with `--skip-check` it prompts:
`Do you want to apply without running the check first? [y/N]`
Probably fine to leave that in, but figured I'd flag.

**Comment by dannyroberts** [comment]
@esoergel Yes, good point. I think being intentional about whether or not that message appears is important, as over-using it and normalizing `y`-ing through those without thinking corrodes its usefulness (a point I articulated when I first noticed it, here: https://github.com/dimagi/commcarehq-ansible/issues/1175).

In this particular case, I considered it and thought it was worth leaving it as a signal to the user that this command goes straight to apply-mode—something that similar-looking commands like `update-config` don't do. I think it's a good cue in this case.
